onmt/dynamic/transforms/tokenize.py

```diff
@@ -0,0 +1,302 @@
+"""Transforms relate to tokenization/subword."""
+from onmt.utils.logging import logger
+from onmt.dynamic.transforms import register_transform
+from .transform import Transform
+
+
+_COMMON_OPT_ADDED = False  # only load common options one time
```

Comment:
We want to put the common options used in various concrete tokenizers in the abstract base `TokenizerTransform` class, and subclass call super().add_options() with it's specific additional options. But as all those Tokenizers will be loaded automatically(except the base as it's abstract hence shouldn't be registered). We wanna only add the common options once to avoid conflict in ArgumentParser, therefore I left this as a flag. Maybe there is a better way.

Reply:
Ok, makes sense. Probably some cleaner way to do this but I don't think of any right now;

Reply:
Find a better solution in #24.